cmd/application.go

```diff
@@ -59,21 +59,19 @@ var getCmd = &cobra.Command{
 var deleteCmd = &cobra.Command{
 	Use:   "delete",
 	Short: "delete the given application",
-	Run: func(cmd *cobra.Command, args []string) {
-		if len(args) == 0 {
-			fmt.Println("Error: specify the application to delete")
-			os.Exit(-1)
-		}
-		if len(args) > 1 {
-			fmt.Println("Error: delete accepts only 1 argument")
-			os.Exit(-1)
+	Args: func(cmd *cobra.Command, args []string) error {
+		if len(args) != 1 {
+			return fmt.Errorf("Please provide application name")
 		}
+		return nil
+	},
+	Run: func(cmd *cobra.Command, args []string) {
 		err := application.Delete(args[0])
 		if err != nil {
 			fmt.Println(err)
 			os.Exit(-1)
 		}
-		fmt.Printf("Deleting application: %v\n", args[0])
+		fmt.Printf("Application %s  was deleted.\n", args[0])
```

Comment:
because deleting means that it is just being deleted, but in the time when this message is showed the application was already deleted (action finished)